pkg/reconciler/subscription/subscription_test.go

```diff
@@ -1424,6 +1421,10 @@ func TestAllCases(t *testing.T) {
 		},
 		{
 			Name: "v1 imc - delivery defaulting - full delivery spec",
+			Ctx: feature.ToContext(context.TODO(), feature.Flags{
+				feature.DeliveryTimeout:    feature.Enabled,
+				feature.DeliveryRetryAfter: feature.Enabled,
+			}),
```

Comment:
Why are we changing existing tests? Can we just add new ones?

Reply:
The testcase in question is `v1 imc - delivery defaulting - full delivery spec` and is focused on testing the defaulting of all the delivery spec values so I just tacked it on there.  I'm happy to break apart the `optional features` into separate test cases though.